payments_service/settings/base.py

```diff
@@ -239,3 +250,9 @@
 SUBSCRIPTION_FROM_EMAIL = 'no-reply@mozilla.com'
 
 MANAGEMENT_URL = 'http://pay.dev:8000/management.html'
+
+# A request origin value that CORS should be enabled for.
+# If this is None, CORS will not be enabled at all.
+ENABLE_CORS_FOR_ORIGIN = ('http://pay.webpack:8080'
+                          if os.environ.get('SERVICE_USE_WEBPACK') == '1'
```

Comment:
Shouldn't this be in the settings file and default to being set by the environment var?

Could it take any boolean?